docs/data-sources/mlflow_models.md

```diff
@@ -0,0 +1,24 @@
+---
+subcategory: "MLflow"
+---
+# databricks_mlflow_models Data Source
+
+-> **Note** This data source could be only used with workspace-level provider!
+
+Retrieves a list of [databricks_mlflow_model](../resources/mlflow_model.md) objects, that were created by Terraform or manually, so that special handling could be applied.
+
+## Example Usage
+
+```hcl
+data "databricks_mlflow_models" "this" {}
+
+output "model" {
+  value = data.databricks_mlflow_models.this
```

Comment:
should we show a more realistic example, like checking if a model exists using `check` block?

e.g. https://developer.hashicorp.com/terraform/language/checks

Reply:
Done